sites/svelte.dev/src/routes/+layout.svelte

```diff
@@ -4,9 +4,21 @@
 	import { Icon, Shell } from '@sveltejs/site-kit/components';
 	import { Nav, Separator } from '@sveltejs/site-kit/nav';
 	import { Search, SearchBox } from '@sveltejs/site-kit/search';
+	import { onNavigate } from '$app/navigation';
 	import '@sveltejs/site-kit/styles/index.css';
 
 	export let data;
+
+	onNavigate((navigation) => {
```

Comment:
Yeah, by default it will do a fade in / fade out. Just pushed a commit to move that to a `/blog` layout so it only applies to navigations under those routes.